wrappers/ios/Sources/Wrapper/Reader/ZXIResult.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 KURZ Digital Solutions GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#import <Foundation/Foundation.h>
+#import "ZXIFormat.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ZXIResult : NSObject
+@property(nonatomic, strong) NSString *text;
+@property(nonatomic, strong) NSData *rawBytes;
```

Comment:
please rename to `binary` for consistency. You seem to need access to 'binary' data? Please have a look at #334 and join the discussion if you find something is missing.